samples/snippets/src/main/java/com/example/monitoring/UptimeSample.java

```diff
@@ -273,10 +268,9 @@ private static void getUptimeCheckConfig(String projectId, String checkName) thr
   // [END monitoring_uptime_check_get]]
 
   // [START monitoring_uptime_check_delete]]
-  private static void deleteUptimeCheckConfig(String projectId, String checkName)
-      throws IOException {
+  private static void deleteUptimeCheckConfig(String checkName) throws IOException {
     try (UptimeCheckServiceClient client = UptimeCheckServiceClient.create()) {
-      client.deleteUptimeCheckConfig(UptimeCheckConfigName.format(projectId, checkName));
+      client.deleteUptimeCheckConfig(checkName);
```

Comment:
When I tried this sample, there was a difference in using the full resource name or ID vs the display name. Can we make it more clear how to get the id vs display name and when they need to be used?

Reply:
I don't see a way to do it using display name anymore. The full check name is in the format of `projects/[PROJECT_ID_OR_NUMBER]/uptimeCheckConfigs/[UPTIME_CHECK_ID]`, where I guess `UPTIME_CHECK_ID` used to be display name, hence the tests were written this way. Now the `UPTIME_CHECK_ID` is display name plus a random string(probably generated in the backend), e.g. `check-cf3bd1` vs `check-cf3bd1-Pn123Zpa-k4`, so all the tests started failing because unable to find the uptime check. The `UPTIME_CHECK_ID` can be retrieved simply by `UptimeCheckConfig#getName()`, I can provide some comments if you think it's necessary.